src/Ledger/PParams.lagda

```diff
@@ -47,65 +46,60 @@ record PParams : Set where
 \emph{Network group}
 \AgdaTarget{maxBlockSize, maxTxSize, maxHeaderSize, maxValSize, pv}
 \begin{code}
-        maxBlockSize       : ℕ
-        maxTxSize          : ℕ
-        maxHeaderSize      : ℕ
-        maxValSize         : ℕ
-        pv                 : ProtVer -- retired, keep for now
+        maxBlockSize maxTxSize        : ℕ
+        maxHeaderSize maxValSize      : ℕ
+        pv                            : ProtVer -- retired, keep for now
 \end{code}
 \emph{Economic group}
 \AgdaTarget{a, b, minUTxOValue, poolDeposit}
 \begin{code}
-        a                  : ℕ
-        b                  : ℕ
-        minUTxOValue       : Coin
-        poolDeposit        : Coin
+        a b                           : ℕ
+        minUTxOValue poolDeposit      : Coin
 \end{code}
 \emph{Technical group}
 \AgdaTarget{Emax, collateralPercent}
 \begin{code}
-        Emax               : Epoch
-        collateralPercent  : ℕ
+        Emax                          : Epoch
+        collateralPercent             : ℕ
 \end{code}
 \emph{Governance group}
 \AgdaTarget{drepThresholds, poolThresholds, ccMinSize, ccMaxTermLength, govActionLifetime, govActionDeposit, drepDeposit, drepActivity, minimumAVS}
 \begin{code}
-        drepThresholds     : DrepThresholds
-        poolThresholds     : PoolThresholds
-        govActionLifetime  : ℕ
-        govActionDeposit   : Coin
-        drepDeposit        : Coin
-        drepActivity       : Epoch
-        ccMinSize          : ℕ
-        ccMaxTermLength    : ℕ
-        minimumAVS         : Coin
+        drepThresholds                : DrepThresholds
+        poolThresholds                : PoolThresholds
+        govActionLifetime             : ℕ
+        govActionDeposit drepDeposit  : Coin
+        drepActivity                  : Epoch
+        ccMinSize ccMaxTermLength     : ℕ
+        minimumAVS                    : Coin
 
-        -- Script
-        -- costmdls            : Language →/⇀ CostModel (Does not work with DecEq)
-        costmdls            : CostModel
-        prices              : Prices
-        maxTxExUnits        : ExUnits
-        maxBlockExUnits     : ExUnits
-        coinsPerUTxOWord    : Coin
-        -- collateralPercent   : ℕ
-        maxCollateralInputs : ℕ
+  -- Script group
```

Comment:
There's no such thing as a script group, see https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694#protocol-parameter-groups

Reply:
Yes, I know there was no such thing as a script group in the cip, but since recently there were a whole bunch of script parameters added, so...

Reply:
The point is that these parameters already have groups, that's why I've linked the CIP. You should sort them into the groups they belong.

Reply:
@WhatisRT Thanks for the clarification.  I've put the new parameters in their rightful group.  As none of these was in the Governance group, am I correct to assume they don't need new documentation, or should I add the documentation of these parameters (from the CIP) to the pdf?